www/js/defaults.json

```diff
@@ -35,6 +35,9 @@
       "autoplay-options": {
         "autoplayTimer": 10
       },
+      "sunder-gutka": {
+        "bani-length": 4
```

Comment:
let's use the same default as the sundar gutka app? I think that one is 2 or 3. cc @maneetpaul @tarunsingh5

Reply:
can be followed up in separate pr

Reply:
The new SG update requires users to choose a length upon the first app launch. Prior to that, the default was "Very Long".

Reply:
maybe we can implement something similar and make it a choose on first install

Reply:
Let's see what the UX is like during Alpha testing and we'll go from there. Bani length only impacts ਬੇਨਤੀ ਚੌਪਈ ਸਾਹਿਬ, ਰਹਰਾਸਿ ਸਾਹਿਬ, ਆਰਤੀ, and ਸੋਹਿਲਾ ਸਾਹਿਬ